renku/command/run.py

```diff
@@ -37,9 +37,14 @@
 from renku.domain_model.provenance.activity import Activity
 
 
-def run_command():
+def run_command(skip_metadata_update: bool):
     """Tracking work on a specific problem."""
-    return Command().command(_run_command).require_migration().require_clean().with_database(write=True).with_commit()
+    command = Command().command(_run_command).require_migration().require_clean()
```

Comment:
I'm a bit unsure about adding it for `renku run`. Since this does not allow using a backend, not updating metadata and not creating a commit is functionally the same as not running with `renku run`. Whereas for the other that execute an already recorded command, there's no alternative way to do that without committing .

Reply:
Ah yeah that is a good point. I will remove it for `renku run` and add it for the other commands you mention.